src/main/kotlin/sc/iview/ui/SwingGroupingInputHarvester.kt

```diff
@@ -61,10 +68,21 @@ class SwingGroupingInputHarvester : SwingInputHarvester() {
     @Parameter
     private lateinit var convertService: ConvertService
 
-
     // -- InputHarvester methods --
+    /**
+     * Builds the Swing panel, with groups defined by the "group:" string in the widget style.
+     */
     @Throws(ModuleException::class)
     override fun buildPanel(inputPanel: InputPanel<JPanel, JPanel>, module: Module) {
+        buildPanel(inputPanel, module, false)
```

Comment:
What is the reason for creating the overloaded function here instead of setting `uiDebug: Boolean = false` per default?